--- a/libgeotiff/CMakeLists.txt
+++ b/libgeotiff/CMakeLists.txt
@@ -80,7 +80,9 @@ SET(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
 
 IF(WIN32)
     IF(MSVC)
-        ADD_DEFINITIONS(-DBUILD_AS_DLL=1)
+        IF(BUILD_SHARED_LIBS)
+            ADD_DEFINITIONS(-DBUILD_AS_DLL=1)
+        ENDIF()
         ADD_DEFINITIONS(/DW4)
         if (NOT (MSVC_VERSION VERSION_LESS 1400))
             ADD_DEFINITIONS(-D_CRT_SECURE_NO_DEPRECATE)
@@ -92,7 +94,7 @@ IF(WIN32)
 ENDIF()
 
 IF(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
-    SET(COMPILE_FLAGS "-fPIC -Wall -Wno-long-long")
+    SET(COMPILE_FLAGS "-Wall -Wno-long-long")
     SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${COMPILE_FLAGS} -std=c99")
     SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${COMPILE_FLAGS} -std=c++98")
     IF(GEOTIFF_BUILD_PEDANTIC)
@@ -171,7 +173,8 @@ IF(WITH_TIFF)
       FIND_PACKAGE(TIFF REQUIRED)
     endif ()
 
-    IF(TIFF_FOUND)
+    # These checks might be too fragile, and we don't care weither HAVE_TIFF is defined or not
+    IF(FALSE)
         # Confirm required API is available
         INCLUDE(CheckFunctionExists)
         SET(CMAKE_REQUIRED_LIBRARIES ${TIFF_LIBRARIES})
@@ -301,7 +304,6 @@ INSTALL(FILES ${GEOTIFF_LIB_HEADERS} DESTINATION include)
 ###############################################################################
 # Build libxtiff library
 
-ADD_SUBDIRECTORY(libxtiff)
 
 ###############################################################################
 # Build libgeotiff library
@@ -363,10 +365,7 @@ SET_TARGET_PROPERTIES(${GEOTIFF_LIBRARY_TARGET} PROPERTIES
    OUTPUT_NAME ${GEOTIFF_LIB_NAME})
 
 TARGET_LINK_LIBRARIES(${GEOTIFF_LIBRARY_TARGET}
-    ${TIFF_LIBRARIES}
-    ${PROJ_LIBRARIES}
-    ${ZLIB_LIBRARIES}
-    ${JPEG_LIBRARIES})
+    ${CONAN_LIBS})
 
 # INSTALL(TARGETS ${GEOTIFF_ARCHIVE_TARGET} ${GEOTIFF_LIBRARY_TARGET}
 #	RUNTIME DESTINATION ${GEOTIFF_BIN_DIR}
